Assets/MRTK/SDK/Experimental/Features/UX/BoundsControl/Visuals/RotationHandles.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.MixedReality.Toolkit.Experimental.UI.BoundsControl
     public class RotationHandles : HandlesBase
     {
         protected override HandlesBaseConfiguration BaseConfig => config;
-        private RotationHandlesConfiguration config;
+        protected RotationHandlesConfiguration config;
```

Comment:
Prepping for extensible/subclassed handles.

Reply:
Thanks for leaving this comments on the intent of these changes - super helpful in reviewing!

Reply:
Reverted, back to `private`.